make/configure.py

```diff
@@ -579,13 +693,13 @@ def _action( self ):
             elif 'powerpc64' in self.mode:
                 del self.mode['powerpc64']
                 self.mode.mode = 'ppc64'
-        elif host.match( '*-*-linux*' ):
+        elif host_tuple.match( '*-*-linux*' ):
             pass
-        elif host.match( '*-*-solaris*' ):
+        elif host_tuple.match( '*-*-solaris*' ):
             pass
-        elif host.match( '*-*-freebsd.*' ):
-            self.mode['i386']   = 'i386-portsbuild-freebsd%s' % (host.release)
-            self.mode['amd64'] = 'amd64-portsbuild-freebsd%s' % (host.release)
+        elif host_tuple.match( '*-*-freebsd.*' ):
```

Comment:
@bradleysepos does this look right to you?  i.e. `*-*-freebsd.*`.  Seems like the '.' shouldn't be there.  I didn't change this, just noticed it while reviewing my own code for typos.

Reply:
Looks like a typo. Feel free to fix.